cmake/thirdparty/SetupSeracThirdParty.cmake

```
@@ -166,6 +166,20 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
         set(CALIPER_FOUND FALSE)
     endif()
 
+    #------------------------------------------------------------------------------
+    # Sundials
+    #------------------------------------------------------------------------------
+    if (SUNDIALS_DIR)
+        # Sundials is already built from MFEM
+        serac_assert_is_directory(VARIABLE_NAME SUNDIALS_DIR)
+        set(SERAC_USE_SUNDIALS ON CACHE BOOL "")
```

This needs a comment about how it is brought in by MFEM, or we need to bring it in ourselves.

---

Is `# Sundials is already built from MFEM` okay? Or should I explain something in more detail?

---

```suggestion
        # Note: Sundials is currently only used via MFEM and MFEM's target contains it's information
        set(SERAC_USE_SUNDIALS ON CACHE BOOL "")
```

Maybe something like this